native/exe/ServiceControlLaunch/ServiceControlLaunch.cpp

```diff
@@ -14,29 +14,31 @@
  * limitations under the License.
  */
 // ServiceControlLaunch.cpp : Defines the entry point for the application.
-//
 
-#include "stdafx.h"
-
-
-// ServiceControlLauncher.cpp : Defines the entry point for the console application.
-//
+/*
+*		*********************************************************
+*		Microsoft Visual Studio project codepage set to MBCS 

```

Comment:
Huh?  How could Java not support being passed Unicode parameters....Java's char primitive is a 16-bit unicode character.

Reply:
Should probably read JavaVMOption doesn't support Unicode

Reply:
From what I could find online, it's due to the way JNI_CreateJavaVM expects its arguments.  Someone here had the same issue https://github.com/LeeKamentsky/python-javabridge/issues/35.

I'll admit I don't have a good grasp of what all is involved but Oracle's documentation says "the option as a string in the default platform encoding".  (https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/invocation.html).

Near as I can tell, it has to do with the way the JVM itself is compiled for Windows.

Reply:
Ahh OK, now I see.  So this comment is just explaining why it's already the way it is and not changing anything. That's fine then.  But please reword it to indicate that the strings passed in the JavaVMOption struct use the platform default character encoding, so they can't be passed as 16-bit Unicode chars.  There's no question you can do other string operations in Java on Unicode chars. :)

Reply:
Done